testsuite/tests/system/messages/mail_templates.yml

```diff
@@ -25,15 +25,15 @@ subject_templates:
       Body: '----==_mimepart_.+Content-Type: text/plain; charset=UTF-8Content-Transfer-Encoding:
       7bitDear <username>,A new application subscribed to the <aplan> plan on the
       <service> service of the <test_group> account.Application details:  Name:
-      API signup  Description: API signupTo view application, follow this link: https://3scale-admin.<threescale_superdomain>/apiconfig/services/[0-9]+/applications/[0-9]+You
+      API signup  Description: API signupTo view application, follow this link: https://3scale-admin.<threescale_superdomain>/p/admin/applications/[0-9]+You
```

Comment:
@mganisin these changes will break the tests for the previous versions of 3scale, how could we proceed here?

Reply:
You could theoretically create a new template file for 2.11 and choose which one to use depending on the tested version. Not sure if it is the preferred solution tho.

Reply:
discussed live and found an agreement